mountpoint-s3/CHANGELOG.md

```diff
@@ -1,4 +1,13 @@
-## Unreleased
+## v1.0.2 (September 22, 2023)
+
+### Breaking changes
+* No breaking changes.
+
+### Other changes
+* New Mountpoint releases are built on CentOS 7 instead of Amazon Linux 2. This lowers the minimum requirement to run Mountpoint to glibc 2.17 or newer. ([#517](https://github.com/awslabs/mountpoint-s3/pull/517))
+* Updated the CRT submodules to the latest releases. This includes bug fixes in aws-c-s3 library such as aws-c-s3#347 and aws-c-s3#346. ([#529](https://github.com/awslabs/mountpoint-s3/pull/529))
```

Comment:
Specifically for CRT updates, I'd lead with what's fixed rather than the fact we updated it. Customers don't care about dependency updates unless they bring some useful new thing, so it's the new thing we should describe rather than the CRT itself.

Reply:
(If we think there's no useful new thing worth mentioning, then we should just drop it)

Reply:
I think we should mention the fix in aws-c-s3#347: Fixed an issue with internal resource cleanup which could lead to Mountpoint hanging after a high number of file uploads.